tasks/facial_landmark_detection/README.md

```diff
@@ -34,7 +34,7 @@ The search also produces a CSV file (search-results-[date]-[time].csv) containin
 To train a particular architecture identified by its ID (e.g., 58626d23) using the entire dataset and more epochs, run the following command:
 
 ```bash
-torchrun --nproc_per_node=4 train.py --data-path $data_dir --output_dir $output_dir --nas_search_archid $arch_id --search_result_csv $csv \
+torchrun --nproc_per_node=4 train.py --data-path $data_dir --output_dir $output_dir --nas_search_archid $arch_id --search_result_csv $csv_path \
```

Comment:
later in this readme it says `you can modify the csv_path variable in the script` but the variable in the `train_candidate_models.py` python script is named  `csv_file`

Reply:
Good point. Now they are consistently "csv_file".